src/shims/ApiCompatBaseline.netfx.netstandard20.txt

```diff
@@ -0,0 +1,164 @@
+Compat issues with assembly mscorlib:
```

Comment:
Also looks like we are going to need to re-enable a number of the partial facade netfx configurations as part of this work.